docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+
+From a technical point of view the [Kubernetes on DCOS](https://github.com/mesosphere/kubernetes-mesos/blob/master/docs/README.md) integration looks as follows:
+
+![Kubernetes-Mesos integration](k8s-on-dcos/k8s-mesos_integration.png)
+
+Comparing the standard Kubernetes deployment with the Kubernetes on DCOS deployment gives the following:
+
+| Aspect                  | Standard Kubernetes | Kubernetes on DCOS |
+| ----------------------- | ------------------- | ------------------ |
+| **multi-service**       | Kubernetes only     | Yes. Kubernetes co-exists with multiple services on the same cluster, including Spark, Cassandra, Marathon, Chronos, Hadoop, Kafka, Jenkins and many others. |
+| **hybrid cloud**        | via workload portability | via workload portability and multi-datacenter scheduling |
+| **business continuity** | fault-tolerance | fault-tolerance, High Availability, SLAs guaranteed |
+| **container**           | Docker | pluggable incl. Docker and cgroups |
+| **scheduler**           | monolithic | pluggable two-level, multi-services resource arbitration |
+| **support**             | community, on a best effort basis| community and premium (SLA-based) through Mesosphere |
+| **security**            | basic | extended |
+
+Note that the Mesosphere DCOS comes in two versions: a **Community Edition** (CE), which is available (today) for cloud-only deployments, and the **Enterprise Edition** (EE), which can be used in a cloud setup, on-premise or in a hybrid setup. Some of the above listed features, incl. premium support, extended security and monitoring are exclusively available through the EE.
+
+## 4. Components
+
+### 4.1  Prerequisites
+
+The only prerequisite to use Kubernetes on DCOS is one of the following environments (physical and/or VM):
+
+1. Instances in a public-cloud, IaaS infrastructure such as [Google Compute](http://cloud.google.com/), AWS [EC2](http://aws.amazon.com/ec2/), and Microsoft [Azure](https://azure.microsoft.com/).
+1. An on-premise cluster of machines.
+1. A combination of 1. and 2., above.
+
+In **any case** above the physical or virtual machines **must not** be pre-installed with a local operating system—such as from the GNU/Linux or Windows family—as during the setup phase Mesosphere DCOS will install [CoreOS](https://coreos.com/). In future, other GNU/Linux flavors will be supported.
+
+### 4.2 Bill of Materials
+
+Core:
+
+* Cloud provisioning (AWS, Azure, GCE)
+* CoreOS
+* Mesos
+* Zookeeper, Exhibitor
+* Mesos-DNS
+* DCOS Distribution Package
+* DCOS Package Repository
+* DCOS CLI
+* DCOS GUI
+
+Included Services:
+
+* Kubernetes
+* Marathon
+* Chronos
+
+Installable Services:
+
+* HDFS
+* Kafka
+* Jenkins
+* Spark
+* Cassandra
+* Hadoop
+* YARN (via Myriad)
+* and many others
+
+## 5. Example Deployment
```

Comment:
This seems to duplicate some of the material from
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/mesos.md

Do we need both?